futures-async-macro/src/lib.rs

```diff
@@ -311,7 +406,7 @@ impl Folder for ExpandAsyncFor {
                             }
                         }
                         futures_await::Async::NotReady => {
-                            yield;
+                            yield futures_await::Async::NotReady;
```

Comment:
I think this may be best using `::futures::...`.

Could a test be added for this as well?

Reply:
There's an `extern crate futures_await;` line just above. This whole `#[async] for` expander uses that instead of via `::futures`.